### PR TITLE
Cow game

### PR DESCRIPTION
**Pull request by anxolin**
Adds a new page with the Cow Game.

The cowgame comes from a company I did in this other project: https://github.com/gnosis/cow-runner-game

image

Adds also a menu item:
image

## Not included
* Some sprites and resources are missing. For example, the sprites from low density of pixels. @michelbio might want to make a new release in this other project https://github.com/gnosis/cow-runner-game adding those. Also final touches. These will be separate PRs.
* Adding a link from the confirmation modal when you trade. Maybe add it also as a banner in recent history.
* ~~The NPM package still points to @anxolin namespace. This is because it's failing for me the deployment in Gnosis NPM organization~~.

@michelbio Maybe we also want to play a bit with the size of the window so is bigger and the sandwitches look better.

## Test
1. Go to
https://pr670--gpswapui.review.gnosisdev.com/#/play
2. Try not to get sandwiched

### CODE REVIEW

File: src/custom/pages/CowGame/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Page, { Title, Content } from 'components/Page'
+import styled from 'styled-components'
+import { CowGame } from '@anxolin/cow-runner-game'
+
+const Wrapper = styled(Page)``
+
+export default function CowGame2() {
```
Comment (alfetopito) [inline src/custom/pages/CowGame/index.tsx:8]: 2?

Comment (anxolin) [reply src/custom/pages/CowGame/index.tsx:8]: 🐮🐮

**Comment by alfetopito** [comment]
The build seems stuck :shrug: 

Whatever is there right now:
1. I still see a T-rex
2. The content position shifts up when the game starts

Start | Running
|-|-|
screenshot_2021-05-26_10-56-06 | screenshot_2021-05-26_10-56-15

File: src/custom/components/Menu/index.tsx [hunk 2]
```diff
@@ -162,6 +162,13 @@ export function Menu() {
           </span>
         </MenuItem>
 
+        <InternalMenuItem to="/play" onClick={close}>
+          <span role="img" aria-label="wizard-icon">
+            🐮
+          </span>
+          Game
+        </InternalMenuItem>
```
Comment (alfetopito) [inline src/custom/components/Menu/index.tsx:170]: Add a space before the text:
screenshot_2021-05-28_08-05-06

**Review by alfetopito — COMMENTED** [review]
I still have the issue of the content moving up when the game starts (see my previous comment)

And we'll probably need to account for dark mode
screenshot_2021-05-28_08-07-10

**Comment by anxolin** [comment]
> I still see a T-rex

You need a laptop with more pixel density :) 

This is not part of this PR. It will come with @michelbio PRs of latest version of the game, with all the sprites

**Comment by anxolin** [comment]
> I still have the issue of the content moving up when the game starts (see my previous comment)

Yes, I'm leaving out the proper layout. I think is how is styled. I would think we will do this later, also making the content bigger.

File: src/custom/components/Menu/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/custom/components/Menu/index.tsx]: done

File: src/custom/pages/CowGame/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Page, { Title, Content } from 'components/Page'
+import styled from 'styled-components'
+import { CowGame } from '@gnosis.pm/cow-runner-game'
+
+const Wrapper = styled(Page)``
```
Comment (W3stside) [inline src/custom/pages/CowGame/index.tsx:6]: why?

File: src/custom/pages/CowGame/index.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        Run{' '}
+        <span role="img" aria-label="cow-icon">
+          🐮
+        </span>{' '}
+        Run!
+      </Title>
+      <p>
+        ...and try not getting{' '}
+        <span role="img" aria-label="sandwich-icon">
+          🥪
```
Comment (W3stside) [inline src/custom/pages/CowGame/index.tsx:21]: would add `'d`, like "try not getting 🥪'd"

File: src/custom/pages/CowGame/index.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        </span>
+        . MEV it&apos;s being lethal these days.
```
Comment (W3stside) [inline src/custom/pages/CowGame/index.tsx:23]:
```suggestion
        . MEV, it&apos;s lethal these days.
```

**Review by W3stside — APPROVED** [review]

**Review by biocom — APPROVED** [review]

**Pull request merged**